bedrock/firefox/templates/firefox/switch.html

```diff
@@ -17,14 +17,19 @@
 
 {% block content %}
 <main>
-  {% call hero(
-    title=ftl('switch-switch-from-chrome'),
-    desc=ftl('switch-switching-to-firefox-page-description-updated', 'switch-switching-to-firefox-page-description'),
-    class='mzp-has-image mzp-t-product-firefox t-switch',
-    include_cta=False,
-    heading_level=1
-  ) %}
-  {% endcall %}
+  {% call split(
+      image_url='img/firefox/switch/switch.png',
+      include_highres_image=True,
+      media_after=True,
```

Comment:
The live version removes the image on smaller screen. Since space is such a premium on mobile, it's probably worthwhile using the `mzp-l-split-hide-media-on-sm-md` class here.

Reply:
ah! good eye